includes/widgets/rating.php

```diff
@@ -217,7 +217,14 @@ protected function register_controls() {
 	}
 
 	protected function get_rating_value(): float {
-		return floatval( $this->get_settings_for_display( 'rating_value' ) );
+		$initial_value = $this->get_rating_scale();
+		$rating_value = $this->get_settings_for_display( 'rating_value' );
+
+		if ( '' === $rating_value ) {
+			$rating_value = $initial_value;
+		}
+
+		return floatval( $rating_value );
```

Comment:
I have changed this because the default schema.org ratingValue was set to 0 in the previous version.